spec/services/akid_parser_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe AkidParser do
+  let(:valid_mailing_id) {'14203'}
+  let(:valid_actionkit_user_id) {'7145943'}
+  let(:valid_akid) {"#{valid_mailing_id}.#{valid_actionkit_user_id}.Si3iNO"}
+  let(:valid_return) { {mailing_id: valid_mailing_id, actionkit_user_id: valid_actionkit_user_id}}
+  let(:invalid_return) { {mailing_id: nil, actionkit_user_id: nil} }
+
+  it 'correctly parses a valid akid' do
+    expect(AkidParser.new(valid_akid).invalid?).to eq false
```

Comment:
I'm on board with that but have different taste/precedent to be consistent with - I think eq is more readable and I use it everywhere except when object identity is actually important, so I'm being self-consistent... but I agree this is fine no matter how we do it.